AzureCalling/Views/ParticipantView.swift

```diff
@@ -37,8 +37,21 @@ class ParticipantView: UIView {
 
     // MARK: Operation methods
 
-    func updateDisplayName(displayName: String) {
-        participantLabel.text = displayName
+    func updateDisplayName(displayName: String, isMuted: Bool) {
```

Comment:
1. Would it be more reasonable that we have a separate api to update muting state icon based on the single responsibility?
2. Instead of appending an NSTextAttachment, we could add an icon in the participantNameLabel, and one separated api interface to toggle the icons change. Since our project is more storyboard based and we are trying to make less UI business code in the code

Reply:
Agreed on the separate api. Made the change to utilize storyboard for UI logic